cfme/tests/containers/test_static_custom_attributes.py

```diff
@@ -14,14 +21,43 @@
     pytest.mark.tier(2)]
 pytest_generate_tests = testgen.generate([ContainersProvider], scope='function')
 
+
+def get_random_string(length):
```

Comment:
It looks like `fauxfactory.gen_alphanumeric()` would be sufficient here? If you require some symbols you can always concatenate those, and you can still replace multiple spaces with a single.  Was there a reason you decided to implement instead of using the built-in?